tests/cryptolibAESSHA/mmcau_des_functions.S

```diff
@@ -0,0 +1,257 @@
+#*******************************************************************************
+#*******************************************************************************
+#
+# Copyright (c) Freescale Semiconductor, Inc 2011.
```

Comment:
under what license does Freescale distribute this assembler file?

Reply:
same question for the remaining `.S` files too

Reply:
Posted the license. It's permissive.